docs/tutorial/cvmfs/what-is-cvmfs.md

```diff
@@ -35,6 +35,8 @@ support this, including:
 * [compression of data](#features-compression);
 * [verification of data integrity](#features-data-integrity);
 
+<span style="color: red;">data vs files? would keep it consistent</span>
```

Comment:
@HereThereBeDragons Well, I guess it depends on the context...

Server-side, "data" makes more sense, since CernVM-FS stores the software installations in a hash tree.
Technically they're just files, but there's "useless" files when accessed directly.

"Files" makes more sense when you're referring to what a CernVM-FS client exposes to end users.